todo_app/app.py

```diff
@@ -1,5 +1,9 @@
+from operator import itemgetter
 from flask import Flask
-
+from flask import render_template
+from flask import request
+from werkzeug.utils import redirect
```

Comment:
```suggestion
from flask import redirect
```
These mostly behave the same, but you might run into issues using the `werkzeug` one if you're looking at the Flask docs!

Also you can import multiple things from one package in a line, e.g.
```python
from flask import Flask, render_template, request
```